ApiReview/API.Polly.RateLimiting/NoDocs/Polly.RateLimiting/ResilienceRateLimiter.cs

```diff
@@ -0,0 +1,13 @@
+// Assembly 'Polly.RateLimiting'
+
+using System.Runtime.CompilerServices;
+using System.Threading.RateLimiting;
+using System.Threading.Tasks;
+
+namespace Polly.RateLimiting;
+
+public sealed class ResilienceRateLimiter
```

Comment:
given that this type accepts a disposable RateLimiter, shouldn't it implement IDisposable?

Reply:
Make `IDisposable`

Reply:
Done.